src/charm.py

```diff
@@ -137,7 +180,10 @@ def _login_ui_layer(self) -> Layer:
                 self._container_name: {
                     "override": "replace",
                     "summary": "identity platform login ui",
-                    "command": "identity_platform_login_ui",
+                    "command": '/bin/sh -c "{} 2>&1 | tee -a {}"'.format(
+                        self._login_ui_service_command,
+                        str(self._log_path),
+                    ),
```

Comment:
Would it make sense to solve the log forwarding problem on the login-ui rather than using this hack?

Reply:
For this, I would review the other Loki Consumer object in the loki charm lib. The current we use which is LogProxyConsumer downloads a promtail binary, and configures it with the given log files.

Reply:
If we all agree on having the login UI write the logs into a file, I'll just change this.